base/applications/calc/winmain.c

```diff
@@ -1945,17 +1945,17 @@ int WINAPI _tWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPTSTR lpCmdL
                     hInstance,
                     MAKEINTRESOURCE(IDI_CALC),
                     IMAGE_ICON,
-                    GetSystemMetrics(SM_CXICON),
-                    GetSystemMetrics(SM_CYICON),
-                    0);
+                    0,
+                    0,
+                    LR_DEFAULTSIZE | LR_SHARED);
 
     calc.hSmIcon = LoadImage(
                     hInstance,
                     MAKEINTRESOURCE(IDI_CALC),
                     IMAGE_ICON,
                     GetSystemMetrics(SM_CXSMICON),
                     GetSystemMetrics(SM_CYSMICON),
```

Comment:
I tend to approve, just one very last question for my understanding, before I'll do:
Why do we use the GetSystemMetrics() calls for the small icon, but do change those to 0 for the large icon a few lines above?

Reply:
Functionally equivalent: `0 and LR_DEFAULTSIZE == SM_CXICON and SM_CYICON`, like other calls which do not needed non-default sizes.

Reply:
Interesting. The version without the GetSystemMetrics()-calls looks cheaper then. Nvm. Approved also in its current state. Don't want to add any risk with something I am not familiar with.